extensions/2.0/Vendor/GRIFFEL_bim_data/schema/glTF.GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data glTF Extension",
+    "type": "object",
+    "description": "Domain specific properties for glTF nodes.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "nodeProperties": {
+            "type": "array",
+            "items": {
+                "$ref": "node.properties.schema.json",
```

Comment:
The referenced schema says that it's used only for separately-stored data.

Reply:
please, describe issue in more details.

Reply:
@lexaknyazev could you clarify if we need to replace the type here or something, probably we'd like to have a call if possible to get an explanation.

Reply:
This line "imports" the `node.properties.schema.json` schema that says "Used only when data is saved in a separate file." in its `description`.

On top of that, there's no full schema for the externally-stored data (as `node.properties.schema.json` is supposed to represent `nodeProperties` elements rather than the full MsgPack payload).